src/chmod/chmod.rs

```diff
@@ -40,7 +38,27 @@ pub fn uumain(args: Vec<String>) -> i32 {
     opts.optflag("R", "recursive", "change files and directories recursively");
     opts.optflag("h", "help", "display this help and exit");
     opts.optflag("V", "version", "output version information and exit");
-    // TODO: sanitize input for - at beginning (e.g. chmod -x testfile).  Solution is to add a to -x, making a-x
+    // sanitize input for - at beginning (e.g. chmod -x testfile). Remove
+    // the option and save it for later, after parsing is finished.
+    let mut negative_option = None;
+    for i in 0..args.len() {
+        if let Some(first) = args[i].chars().nth(0) {
```

Comment:
You can change `Some(first)` to `Some('-')` here so you can get rid of the check below.